tests/features/seo/sitemap.feature

```diff
@@ -0,0 +1,8 @@
+@openy @api
+Feature: Sitemap
+  Ensure that sitemap is available and a landing page is present.
+
+  Scenario: Check sitemap
+    Given I am an anonymous user
+    When I go to "/sitemap.xml"
+    Then I should see text "blog/mango-avocado-salsa" in XML
```

Comment:
We cannot rely on demo content in tests.
Here's an example how to generate blog post in behat in Background https://github.com/ymcatwincities/openy/blob/8.x-1.x/tests/features/content_types/blog_post.feature#L5